lib/generators/react_on_rails/templates/base/base/client/webpack.config.js

```diff
@@ -5,9 +5,9 @@
 const webpack = require('webpack');
 const { resolve } = require('path');
 const ManifestPlugin = require('webpack-manifest-plugin');
-const webpackConfigLoader = require('react-on-rails').default.webpackConfigLoader; // eslint-disable-line import/no-unresolved
-
-const { env, paths, publicPath } = webpackConfigLoader(resolve('..', 'config', 'webpack'));
+const webpackConfigLoader = require('react-on-rails/node_package/lib/webpackConfigLoader').default;
+const configPath = resolve('..', 'config', 'webpack');
+const { env, paths, publicPath } = webpackConfigLoader(configPath);
```

Comment:
@alexfedoseev and @robwise I have no idea on what you're referring to. The only reason to modify the webpack configs is so that the webpacker_lite view helper will be able to go the public output directory and thus bypass the asset pipeline.